rs-libc/src/c/floatscan.c

```diff
@@ -418,7 +417,9 @@ static long double hexfloat(FILE *f, int bits, int emin, int sign, int pok)
 	return scalbnl(y, e2);
 }
 
+#ifndef _MSC_VER
 __attribute__ ((visibility ("hidden")))
```

Comment:
Yeah I think unfortunately there is just no equivalent of this in MSVC. So this ifdef is fine.